app/controllers/application_controller.rb

```diff
@@ -996,7 +996,8 @@ def view_to_hash(view, fetch_data = false)
         when "result"
           new_row[:cells] << {:span => result_span_class(row[col]), :text => row[col].titleize}
         when "severity"
-          new_row[:cells] << {:span => severity_span_class(row[col]), :text => row[col].titleize}
+          value = row[col] || ' '
+          new_row[:cells] << {:span => severity_span_class(value), :text => severity_title(value)}
```

Comment:
we need to move these to some separate class,

and do the condition on **report name** + column name, not just column names